draft-gpew-priv-ppm.md

```diff
@@ -317,19 +319,26 @@ until the entire batch of reports is received.
 
 ## Validating Inputs {#validating-inputs}
 
-An essential task of any data collection pipeline is ensuring that the input
-data is "valid". In PPM, input validation is complicated by the fact that none
-of the entities other than the client ever sees the values for individual
-clients.
+An essential task of any data collection pipeline is ensuring that the data
+being aggregated is "valid". In PPM, input validation is complicated by the fact
+that none of the entities other than the client ever sees the values for
+individual clients.
 
 In order to address this problem, the aggregators engage in a secure,
-multi-party computation, specified by the chosen VDAF, to ensure that consuming
-the report will not corrupt the aggregate result. The input shares
-generated by the client include information used by the aggregators during
-aggregation. For example, `prio3` includes a distributed zero-knowledge proof of
-the input's validity [BBCGGI19] which the aggregators can jointly verify
-and reject the report if it cannot be verified. However, they do not
-learn anything about the individual report other than that it is valid.
+multi-party computation specified by the chosen VDAF
+{{?I-D.draft-cfrg-patton-vdaf}} in order to prepare a report for aggregation. At
+the beginning of this computation, each aggregator is in possession of an input
+share uploaded by the client. At the end of the computation, each aggregator is
+in posession of either an "output share" that is ready to be aggregated or an
+indication that a valid output share could not be recovered.
+
+To facilitiate this computation, the input shares generated by the client
+include information used by the aggregators during aggregation in order to
+validate their recovered output shares. For example, `prio3` includes a
+distributed zero-knowledge proof of the input's validity [BBCGGI19] which the
```

Comment:
```suggestion
distributed zero-knowledge proof of the input's validity {{BBCGGI19}} which the
```

Reply:
Hmm, the tool that renders the markdown into XML seems to interpret this correctly. Plus we've been using `{{XXX}}` rather than `{{blah}}` for paper references fairly consistently throughout the doc. Do you think, we should change this back?

Reply:
`[..]` does nothing in the translation process. It literally duplicates the text inline. You could write `[bananas]` and it would render `[bananas]` in the XML/TXT/HTML output. `{{...}}`` actually invokes the translation process to cite things correctly. So, yeah, we should be using `{{...}}`.

Reply:
That's not true, as far as I can tell. take a look at the rendered HTML. A link is put there that navigates to the informational references at the bottom. And we don't use the `{{...}}` syntax anywhere for this particular reference.

Reply:
As discussed offline, I'll follow up with a PR that changes all the [...] style references with {{...}} style references.